src/Playground.res

```diff
@@ -1315,9 +1315,7 @@ module InitialContent = {
 }
 `
 
-  let since_10_1 = `@@jsxConfig({ version: 4, mode: "automatic" })
-
-module CounterMessage = {
+  let since_10_1 = `module CounterMessage = {
```

Comment:
This is not straightforward for a beginner to understand. I don’t think it introduces any regressions, but could you confirm that?

Reply:
Yes, remove it.